test/powershell/Modules/Microsoft.PowerShell.LocalAccounts/Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1

```diff
@@ -0,0 +1,853 @@
+﻿# This is a Pester test suite to validate the cmdlets in LocalAccounts module
+#
+# Copyright (c) Microsoft Corporation, 2015
+
+# LocalAccounts does not work outside of windows
+if(-not $IsWindows)
+{
+    return
+}
+
+function RemoveTestGroups
+{
+    param([string] $basename)
+    
+    $results = Get-LocalGroup $basename* 
+    foreach ($element in $results) {
+        Remove-LocalGroup -SID $element.SID
+    }
+}
+
+function VerifyFailingTest
+{
+    param(
+        [scriptblock] $sb,
+        [string] $expectedFqeid
+    )
+
+    $backupEAP = $script:ErrorActionPreference
+    $script:ErrorActionPreference = "Stop"
+
+    $actualFqeid = ""
+    try { 
+        & $sb
+    } 
+    catch { 
+        $actualFqeid = $_.FullyQualifiedErrorId 
+    } 
+    finally {
+        $script:ErrorActionPreference = $backupEAP
+    }
```

Comment:
It's recommended to follow the `try/catch` pattern described in https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#writing-pester-tests